### PR TITLE
sequence: add AF job to import HAR sequence

### PR DESCRIPTION
**Pull request by thc202**
Allow to create Sequence from HAR file.

### CODE REVIEW

File: addOns/sequence/src/main/javahelp/org/zaproxy/zap/extension/sequence/resources/help/contents/automation.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<br><br>
+
+<H2>Job: sequence-import</H2>
+The <code>sequence-import</code> job allows you to create a Sequence from an HAR file.
+<pre>
+  - type: sequence-import      # Imports a sequence from a HAR file.
+    parameters:
+      name:                    # The name by which the seq will be known in ZAP.
+      path:                    # The full/relative path to the HAR file to import.
+      assertCode:              # Boolean, if true add code assertion.
```
Comment (kingthorin) [inline addOns/sequence/src/main/javahelp/org/zaproxy/zap/extension/sequence/resources/help/contents/automation.html:22]: Should the comment (if not also the field name) be more clear this is status code or response code?

File: addOns/sequence/src/main/resources/org/zaproxy/zap/extension/sequence/resources/Messages.properties [hunk 2]
```diff
@@ -1,3 +1,19 @@
+sequence.automation.desc = Sequence Automation Framework Integration
+sequence.automation.dialog.jobName = Job Name:
+sequence.automation.error.noresourcefile = Cannot access file: {0}
+sequence.automation.import.dialog.assertCode = Assert Code:
```
Comment (kingthorin) [inline addOns/sequence/src/main/resources/org/zaproxy/zap/extension/sequence/resources/Messages.properties:4]: Same here re: status or response

File: addOns/sequence/src/main/javahelp/org/zaproxy/zap/extension/sequence/resources/help/contents/automation.html (hunk 1 above)
Comment (thc202) [reply addOns/sequence/src/main/javahelp/org/zaproxy/zap/extension/sequence/resources/help/contents/automation.html]: Updated the comment.

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Still LGTM

**Review by psiinon — APPROVED** [review]

**Pull request merged**